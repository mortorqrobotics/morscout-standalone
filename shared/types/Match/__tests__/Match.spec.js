import { loadMatch, updated } from "../index";

test("Load Match stays the same", () => {
  expect(loadMatch).toMatchSnapshot();
});

test("Updated Match stays the same", () => {
  expect(updated).toMatchSnapshot();
});
