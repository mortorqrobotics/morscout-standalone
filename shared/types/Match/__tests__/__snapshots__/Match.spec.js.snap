// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Load Match stays the same 1`] = `"server/LOAD_MATCH"`;

exports[`Updated Match stays the same 1`] = `"UPDATED_MATCH"`;
